internal/storage/v2/clickhouse/config/config.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"github.com/jaegertracing/jaeger/internal/storage/v2/clickhouse/client/conn"
+	"github.com/jaegertracing/jaeger/internal/storage/v2/clickhouse/client/pool"
+)
+
+// Configuration all configuration of how to use clickhouse as storage.
+// ConnConfig clickhouse-go client configuration
+// PoolConfig chpool configuration
+// CreateSchema create requre table auto if not exist.
+type Configuration struct {
+	ConnConfig   conn.Configuration
```

Comment:
please keep all configs together, there's no reason for them to be in different packages